app/grandchallenge/core/utils/markdown.py

```diff
@@ -41,3 +43,29 @@ def run(self, root):
             if el.tag == "a":
                 el.set("target", "_blank")
                 el.set("rel", "noopener")
+
+
+class HtmlTagsExtension(Extension):
+    def extendMarkdown(self, md):  # noqa: N802
+        md.registerExtension(self)
+        md.postprocessors.register(
+            HtmlTagsPostprocessor(md), "htmltags_extension", 35
```

Comment:
```suggestion
            HtmlTagsPostprocessor(md), "html_tags_extension", 35
```

Missing underscore. What does 35 mean?

Reply:
The number is the priority of the processor (order of execution, higher priority is earlier execution). In the package they use three levels of priority with steps of 10, so 30 is the highest, and smaller increments by 5 for in between. In the postprocessor level, the [RawHtmlPostprocessor](https://python-markdown.github.io/extensions/api/#usages_4) is used to inject back the HTML blocks after processing the markdow, so we want our processor (adding the classes to the HTML tag) to execute before the blocks are injected back into the output text, so I added an increment of 5 to the highest priority 30, so it is 35.

Reply:
This needs a comment, especially if ordering is important. What happens when the priorities upstream change? If this needs to be done before anything else then the priority needs to be automatically assigned from the things it needs to run before.

Reply:
I changed the logic and moved the htmlStash processing to the BS4Treeprocessor which is executed always before the postprocessors, hence, no need for specific priority checkup. But still, the package does handle the raw html the same way as the converted markdown to html and thus they still need to be handles separately. I implemented a solution, check the comment on the review below.